src/sparseml/onnx/utils/helpers.py

```diff
@@ -1233,3 +1234,25 @@ def set_tensor_dim_shape(tensor: onnx.TensorProto, dim: int, value: int):
     :param value: new shape for the given dimension
     """
     tensor.type.tensor_type.shape.dim[dim].dim_value = value
+
+
+def override_model_input_shape(model: Union[str, onnx.ModelProto], shape: List[int]):
+    """
+    Set the shape of the first input of the given model to the given shape.
```

Comment:
can we be more specific that this only works for cv-like models that assume input BCHW?

Reply:
this works for any onnx model, no? shape is generic